tests/layout-checks.py

```diff
@@ -175,7 +184,7 @@ def check_empty_check():
         # z/tempest is a fake project with no check queue
         if project['name'] == 'z/tempest':
             continue
-        check_jobs = collect_pipeline_jobs(project, 'check')
+        check_jobs = collect_pipeline_jobs(project, 'check_github')
```

Comment:
How would that work for example with a gitlab or other provider rather than just gerrit?